app/views/components/_ingredient_item.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="c-ingredient-item">
+  <%= render "components/attached_img", 
+    image: ingredient.food.image, 
+    image_class: "c-ingredient-item__img",
+    image_alt: "ingredient photo",
+    placeholder: "food_placeholder.png"
+  %>
+  <div class="c-ingredient-item__info">
+    <h3 class="c-ingredient-item__title">
+      <%= ingredient.food.name %>
+    </h3>
+    <p class="c-ingredient-item__price">
+      Total price: $<%= ingredient.food.price * ingredient.quantity %>
+    </p>
+  </div>
+  <span class="c-ingredient-item__quantity">
+    <%= "#{ingredient.quantity} #{ingredient.food.measurement_unit}" %>
+  </span>
+
```

Comment:
[REQUIRED] According to the wireframe quantity should come before price
<img width="1440" alt="image" src="https://github.com/IvonneBenitesRodriguez/recipe_app/assets/62938735/fcb0f0be-0339-425e-b8c7-a9a1c3917fd0">